Sources/CollectionView.swift

```diff
@@ -112,11 +119,12 @@ open class CollectionView: UIScrollView {
   public func reloadData(contentOffsetAdjustFn: (() -> CGPoint)? = nil) {
     guard !reloading else { return }
     provider.willReload()
+    flattenedProvider = provider.flattenedProvider()
```

Comment:
here `flattenedProvider` can't be used? Do we need an explicitly assignment here?